src/test/java/org/jenkinsci/plugins/docker/commons/credentials/DockerServerCredentialsTest.java

```diff
@@ -98,7 +99,13 @@ public void configRoundTripUpdateCertificates() throws Exception {
             button.click();
         }
 
-        form.getTextAreaByName("_.clientKeySecret").setText("new key");
+        if (j.jenkins.getVersion().isOlderThan(new VersionNumber("2.413"))) {
```

Comment:
The problem is not here but in line 98 where the button is no longer selected and clicked
It should be changed in the same way that I changed it in core by using
`form.querySelector(".secret-update-btn")` to make it independent if it is an input or a button

Reply:
Thanks very much!  I've updated in c30a5a8b92f15f54196ed73647690505eb3fc26b with the correction.